docs/03-assertions.md

```diff
@@ -203,6 +203,10 @@ Assert that `value` is not the same as `expected`. This is based on [`Object.is(
 
 Assert that `value` is deeply equal to `expected`. See [Concordance](https://github.com/concordancejs/concordance) for details. Works with [React elements and `react-test-renderer`](https://github.com/concordancejs/react).
 
+### `.like(value, likePattern, message?)`
+
+Assert that `value` is like `likePattern`. Comparison is the same as in `.deepEqual()` except only paths present in the `likePattern` object are compared. All paths not in `likePattern` are ignored and can hold any value. Use this if you want to test for some keys and ignore the others.
```

Comment:
It's unclear what a "path" is.

Reply:
A series of keys to address a deeper property of an object. I can change it to "keys", this will be easier to understand but maybe not as accurate. Should I change this?

Reply:
We should describe the algorithm here.